tests/FSharpLint.Core.Tests/Rules/Conventions/SourceLength.fs

```diff
@@ -230,8 +236,8 @@ type TestMaxLinesInClass() =
         this.Parse(sprintf """
 module Program
   type MyClass2() as this =
-    %s
-    member this.PrintMessage() = ()""" (generateNewLines ClassLength))
+    do%s
```

Comment:
@Mersho please put the %s on the next line and make generateNewLines NOT start with an EOL